lib/i18n/traditional-chinese.js

```diff
@@ -0,0 +1,107 @@
+// -*- mode: js; indent-tabs-mode: nil; js-basic-offset: 4 -*-
+//
+// This file is part of Genie
+//
+// Copyright 2018-2019 The Board of Trustees of the Leland Stanford Junior University
```

Comment:
You don't work for Stanford, so the copyright should be NTU?

Reply:
That's right. Thank you!